Default branch tests' failing - Looks like a new ANTs' version may have been released recently

### What happened?

```
  E                       FileNotFoundError: No such file or directory '['/tmp/tmp4m3y60tu/ants-000040GenericAffine.mat']' for output 'reverse_forward_transforms' of a Registration interface
```

### What command did you use?

```shell
pytest
```

### What version of the software are you running?

latest

### How are you running this software?

Local installation ("bare-metal")

### Is your data BIDS valid?

Yes

### Are you reusing any previously computed results?

No

### Please copy and paste any relevant log output.

```shell

```

### Additional information / screenshots

_No response_

--- a/test/test_estimator.py
+++ b/test/test_estimator.py
@@ -103,11 +103,18 @@ def test_estimator_init_model_instance(request):
 
 def test_estimator_init_model_string(request, monkeypatch):
     rng = request.node.rng
+
     # Patch ModelFactory.init to return DummyModel
     monkeypatch.setattr(
         "nifreeze.model.base.ModelFactory.init",
         lambda model, dataset, **kwargs: DummyModel(dataset=dataset),
     )
+
+    def mock_iterator(*_, **kwargs):
+        return []
+
+    monkeypatch.setattr(iterators, "random_iterator", mock_iterator)  # Avoid iterator issues
+
     model_name = "dummy"
     est = Estimator(model=model_name, model_kwargs={})
     _dataset = DummyDataset(rng)

